src/jit/flowgraph.cpp

```diff
@@ -8249,7 +8253,7 @@ void                Compiler::fgAddInternal()
 #if FEATURE_FIXED_OUT_ARGS
         // Grab and reserve space for TCB, Frame regs used in PInvoke epilog to pop the inlined frame.
         // See genPInvokeMethodEpilog() for use of the grabbed var.
-        if (compJmpOpUsed)
+        if (!usePInvokeHelpers && compJmpOpUsed)
```

Comment:
It would be a good idea to add a comment why we are excluding lvaPInvokeFrameRegSaveVar  for Core RT.